features/login.feature

```diff
@@ -48,10 +48,10 @@ Feature: Log in as an admin, as a teacher, as a student
         As a student, I want to be able to login
         When student logs in with email <studentUsername> and password <password>
         And student with full age accepts student's data protection with password <newStudentPassword>
-        Then user-dashboard should have the correct initials
-        Then user-dashboard should have the correct school
-        Then user-dashboard should have the correct name and profession
```

Comment:
why is not yet correctly implemented, because in this step we check full name and role (profession) of user

Reply:
the method behind "user-dashboard should have the correct name and profession" did the same as "user-dashboard should have the correct initials"

`return DashboardPage.loginInitials();`

Reply:
loginInitals() checks only initals "TT" for Thorsten Test
DashboardPage.loginFullUserInfo(); checks full name Thorsten Test (Administrator) and compare with results from API
those steps are different